docs/learn/ticks-and-update-rates.md

```diff
@@ -1,58 +1,39 @@
 
-See [Latency](lagandpacketloss.md#latency)
+Tick rate is measured in hertz (Hz). For example, a tick rate of 60 Hz means the server updates the game state sixty times a second, whereas a tick rate of 30 Hz means the server updates the game states only thirty times a second. Higher tick rates produce a more responsive game experience, at the cost of increased load on the simulating server.
 
-## Update Rate
+When a server fails to process all incoming client data before the end of the current tick, it produces gameplay issues such as rubber banding, player teleporting, hits getting rejected, and physics failing. As such, there's a balance between having a high tick rate to create responsive gameplay and ensuring that the server has enough time to process all client data before the end of a tick.
 
-What adds an extra delay on top of the travel time of our data (ping), is how often a game sends and receives that data. When a game sends and receives updates at 30Hz (30 updates per second), then there is more time between updates than when it sends and receives updates at 60Hz.
+## Update rate
 
-Update Rates, additional delay:
+Update rate is a measure of how frequently the client sends and receives data to and from the server. It's also measured in hertz and, like tick rate, a higher update rate results in a more responsive game at the cost of increased processing and network demands on the client and server.
 
-<ImageSwitcher 
+<ImageSwitcher
 lightImageSrc="/update-rates-light.png?text=LightMode"
 darkImageSrc="/update-rates-dark.png?text=DarkMode"/>
 
-Low update rates don't only affect the network delay; they also cause issues like "super bullets," where a single hit from a gun deals more damage than it should be able to deal. 
-
-For example, a game server sends 10 updates per second. At this update rate we have 100ms between the updates, which is the same time that we have between two bullets when a gun fires 600 rounds per minute.
+In addition to adding to perceived latency, low update rates can cause their own issues such as 'super bullets', where multiple updates are bundled together and arrive at the same time, causing undesirable behavior.
 
-But many shooters have guns which fire 750 rounds per minute or more. As a result, we then run out of updates and so the game has to send multiple bullets with one update. If two bullets hit a player, then the damage of these two hits will be sent in a single update, and so the receiving player will get the experience that he got hit by a "super bullet" that dealt more damage than a single hit should be able to deal.
+For example, if an in-game gun is capable of shooting more times per second than the client can update, then at some point the impact of more than one bullet will be communicated in the same update. For players receiving this update, it will seem as if a single bullet is doing more damage than it should, because they're actually receiving updates from multiple bullets at once.
```

Comment:
If this is the description of a super bullet, I'd move that "such as super bullets" from the previous sentence to the start of this one. 

Unless all instances where multiple updates are bundled together are referred to as super bullets?